styles/variables-silicon.less

```diff
@@ -127,7 +127,9 @@
 
 // Picker
 // ---------------------------------------
-@agate-picker-font-weight: 500;
+@agate-picker-font-size: 28px;
```

Comment:
If possible, we want to use a multiple of 3. Please check https://github.com/enactjs/agate/pull/253#discussion_r431506007 discussion.